runtime/src/lib.rs

```diff
@@ -304,13 +304,19 @@ impl fp_rpc::ConvertTransaction<opaque::UncheckedExtrinsic> for TransactionConve
 
 pub struct EthereumFindAuthor<F>(PhantomData<F>);
 
+parameter_types! {
+	// TODO:
+	pub BlockGasLimit: U256 = U256::from(u32::max_value());
```

Comment:
I thought Joshy worked on that

Reply:
```suggestion
	pub BlockGasLimit: u64 = NORMAL_DISPATCH_RATIO * MAX_BLOCK_WEIGHT / WEIGHT_PER_GAS;
```

I did, but I closed the PR because I got confused about the git history. https://github.com/PureStake/moonbeam/pull/203

Now that Frontier is updated it should be just that line.

Reply:
:+1:

Reply:
Please review 947b223